src/util/make-knex.js

```diff
@@ -8,9 +8,8 @@ import QueryInterface from '../query/methods';
 import * as helpers from '../helpers';
 import { assign } from 'lodash'
 import BatchInsert from './batchInsert';
-import Knex from "../index";
 
-export default function makeKnex(client) {
+export default function makeKnex(client, version) {
```

Comment:
Hmmm... Are we sure this will work correctly? I'm quite certain `makeKnex` is called from more places than just `src/index.js` and those places would then need to be updated as well or they'll set the created `knex` instance's `VERSION` attribute to `undefined`.

I'm not quite sure - why is it that we want to add this version parameter anyway? Can we instead just move the whole version code into a separate module and then include it from there to both here and the code that sets it on the `Knex` class?